### ISSUE TITLE
Generate home link is greyed out when using README.md as index filename.

### ISSUE THREAD

**Issue by chakrit** [mkdocs/mkdocs#1609]
![2018-08-27 at 5 20 pm](https://user-images.githubusercontent.com/45545/44654919-94aded00-aa1d-11e8-9096-95dc16cc14b5.png)

The `Home` link does not work, **it is greyed out** if I opted to use `README.md` as filenameinstead of the default `index.md` because I wanted docs to work vanilla when browsed on github as well.

---

OS: Latest OSX

```
    » pipenv run pip --version
pip 18.0 from /Users/chakrit/.local/share/virtualenvs/documentation-Vk2LaSt9/lib/python3.7/site-packages/pip (python 3.7)

    » pipenv run python --version
Python 3.7.0

    » pipenv graph
mkdocs-cinder==0.12.2
mkdocs-material==3.0.3
  - mkdocs [required: >=1, installed: 1.0]
    - click [required: >=3.3, installed: 6.7]
    - Jinja2 [required: >=2.7.1, installed: 2.10]
      - MarkupSafe [required: >=0.23, installed: 1.0]
    - livereload [required: >=2.5.1, installed: 2.5.2]
      - six [required: Any, installed: 1.11.0]
      - tornado [required: Any, installed: 5.1]
    - Markdown [required: >=2.3.1, installed: 2.6.11]
    - PyYAML [required: >=3.10, installed: 3.13]
    - tornado [required: >=5.0, installed: 5.1]
  - Pygments [required: >=2.2, installed: 2.2.0]
  - pymdown-extensions [required: >=4.11, installed: 5.0]
    - Markdown [required: >=2.6.10, installed: 2.6.11]
```

**Comment by chakrit** [comment]
Contents of mkdocs.yml:

```
---
site_name: Systems Documentation
theme: readthedocs
markdown_extensions:
  - admonition
nav:
  - Home: 'README.md'
  - Services: 'Services.md'
  - Payments: 'Payments.md'
```

**Comment by waylan** [comment]
You appear to have 1.0 installed. This was fixed in 1.0.1. I would suggest you upgrade:

    pip install -U mkdocs
